README.md

```diff
@@ -0,0 +1,354 @@
+Introduction
+------------
+
+Provides a Web server for symbolicating Firefox stacks. It matches PC addresses
+to modules in memory and looks up the corresponding function names in
+server-side symbol files (.SYM files).
+
+This project is intended as a drop-in replacement for
+[Snappy Symbolication Server](https://github.com/mozilla/Snappy-Symbolication-Server/)
+
+Quick Start Guide
+-----------------
+
+1. Install [Python 2.7](https://www.python.org/downloads/) if it is not already
+   installed
+2. Install dependencies:
+   `python -m pip install tornado futures python-memcached psutil`
```

Comment:
Instead of doing this, include a `requirements.txt` file with the following content:

```
tornado
futures
python-memcached
psutil
```

Then change this step to "Install dependencies: pip install -r requirements.txt". That's a better way to package things in Python.